addons/sale_project/models/project.py

```diff
@@ -566,7 +566,7 @@ def action_project_sharing_view_so(self):
         if not self.display_sale_order_button:
             return {}
         return {
-            "name": "Portal Sale Order",
+            "name": _("Portal Sale Order"),
```

Comment:
IMO, it is not really useful since this name is not really rendered in the UI. 🤔

Reply:
Done
Thanks :smile:

Reply:
Thanks 🙂